src/camvidlog2/cli.py

```
@@ -77,12 +78,16 @@ def query(
     if json:
         if queries:
             raise ValueError("Cannot use --json and [QUERIES]")
-        embedding_group = load_embedding_group_json(json)
+        embedding_collection = load_embedding_json(json)
     else:
         if not queries:
             raise ValueError("Must provide either --json or [QUERIES]")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)